src/+equations/calc_rssrbs.m

```diff
@@ -1,5 +1,13 @@
-function [rss, rbs] = calc_rssrbs(SMC, LAI, rbs)
-    global SaturatedMC ResidualMC fieldMC
+function [rss, rbs] = calc_rssrbs(SMC, LAI, rbs, ResidualMC, fieldMC)
+    %{
+    This function is to calculate surface resistance, following C. van der Tol
+    et al. 2014 (BG). The emperical parameters "aa, 4.1, and 4.3" are
+    user-defined. And these parameters can be calibrated during winter season
+    when LAI is low.
+    %}
     aa = 3.8;
-    rss = exp((aa + 4.1) - aa * (SMC - ResidualMC(1)) / (fieldMC(1) - ResidualMC(1)));
```

Comment:
hI Bart, indeed, the index should be there, because this is for surface resistance, so, only surface ResidualMc and field MC  should be used here. I will re-submit the pull request after correcting this.

Reply:
@BSchilperoort Bart, the plot you used is very useful to identify if the results are the same or not. Perhaps we should put such a script of plotting differences into the test workflow?

Reply:
@yijianzeng The notebook I used to generate those plots is in https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/main/test/compare_git_branch_results.ipynb
It allows you to compare the results of two git branches of STEMMUS_SCOPE